### PR TITLE
PHPCS fixes & Optimized code by removing unwanted comments & code.

### PR DESCRIPTION
**Pull request by maheshwaghmare**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+variables:
```
Comment (Nikschavan) [inline .gitlab-ci.yml]: No need of this file here on Github.

File: class-astra-theme-customizer-reset.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		 *
+		 * @since 1.0.0
+		 * @return void
+		 */
+		public function ajax_customizer_reset() {
+
+			// Validate nonce.
+			check_ajax_referer( 'astra-customizer-reset', 'nonce' );
+
+			// Reset option 'astra-settings'.
```
Comment (Nikschavan) [inline class-astra-theme-customizer-reset.php]: Along with Ajax referrer, we should also check if the user has capability.

File: class-astra-theme-customizer-reset.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Plugin Name: Astra Customizer Reset
+ * Plugin URI: http://www.ultimatebeaver.com/
```
Comment (Nikschavan) [inline class-astra-theme-customizer-reset.php]: Change this to Astra's website - https://wpastra.com/

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Ignore files.
+.gitlab-ci.yml
```
Comment (Nikschavan) [inline .gitignore]: @maheshwaghmare There is actually no need of the file.s mention here as well.

File: .gitlab-ci.yml (hunk 1 above)
Comment (maheshwaghmare) [inline .gitlab-ci.yml]: Removed `.gitlab-ci.yml` file.

File: class-astra-theme-customizer-reset.php (hunk 2 above)
Comment (maheshwaghmare) [inline class-astra-theme-customizer-reset.php]: Added `$this->wp_customize->is_preview()` condition before removing customizer options.

File: .gitignore (hunk 4 above)
Comment (maheshwaghmare) [inline .gitignore]: Removed `.gitignore` & other unwanted files.

File: readme.txt [hunk 5]
```diff
@@ -0,0 +1,27 @@
+=== Astra Customizer Reset ===
+Contributors: brainstormforce
+Tags: astra theme, customizer reset, astra customizer reset
+Tested up to: 4.7.3
+Stable tag: 1.4.9
```
Comment (Nikschavan) [inline readme.txt]: Why stable tag `1.4.9`?

Comment (maheshwaghmare) [inline readme.txt]: Now used `4.6` I'll update it.

**Pull request merged**